src/scripts/install_bolt.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/sh
-wget --no-check-certificate https://storage.googleapis.com/broad-alkesgroup-public/BOLT-LMM/downloads/old/BOLT-LMM_v2.3.5.tar.gz && \
-tar -xvzf BOLT-LMM_v2.3.5.tar.gz && \
-rm -rf BOLT-LMM_v2.3.5.tar.gz && \
-mv BOLT-LMM_v2.3.5/* . && \
-rmdir BOLT-LMM_v2.3.5
 
-# NB! Bolt-LMM must be executed from /tools/bolt/bolt because it links to binaties in /tools/bolt/lib folder.
+export VERSION="v2.3.5"
+                           
+wget --no-check-certificate https://storage.googleapis.com/broad-alkesgroup-public/BOLT-LMM/downloads/old/BOLT-LMM_$VERSION.tar.gz
+tar -xvzf BOLT-LMM_$VERSION.tar.gz
+rm -rf BOLT-LMM_$VERSION.tar.gz
+mv BOLT-LMM_$VERSION/* .
+
+# delete some stuff that should not be containerized:
+rm -rf BOLT-LMM_$VERSION
+rm license.txt
```

Comment:
Is it fine to ``rm license.txt``? This is probably a standard copy of the GPLv3, which mainly governs source code distribution, but not so much the binary distribution (at least that is as far as I can understand all of these Open Source licensing stuff). Yet to be safe I'd probably keep the license - though I also don't feel strongly against removing it.

Reply:
It's GPLv3 in this case plus MIT for NLopt which is included in the build. 
Perhaps we can document the license information for each included software in the `src/README` instead?

Reply:
Btw, I messed around a bit with building BOLT-LMM using system libraries (libboost, etc.), but the resulting container size got pretty big (>1GB), so probably not something we should do unless someone reports a problem.